### PR TITLE
fatfs file create error fix

### PR DESCRIPTION
**Pull request by Deryugin**
Fix error on file create. Error was related to ramdisk block size (default 4096, and fat was expecting 512).

### CODE REVIEW

File: src/drivers/block_dev/ramdisk/ramdisk.c [hunk 1]
```diff
@@ -167,7 +166,7 @@ static int read_sectors(block_dev_t *bdev,
 	char *read_addr;
 
 	ramdisk = (ramdisk_t *) bdev->privdata;
-	read_addr = ramdisk->p_start_addr + (blkno * ramdisk->block_size);
+	read_addr = ramdisk->p_start_addr + (blkno * bdev->block_size);
```
Comment (alexkalmuk) [inline src/drivers/block_dev/ramdisk/ramdisk.c]: It seems something is wrong with ramdisk->block_size. If I understand right ramdisk->block_size and bdev->block_size mean the same thing. So may be remove the first one at all?

**Comment by alexkalmuk** [comment]
LGTM

**Comment by anton-bondarev** [comment]
LGTM

File: src/drivers/block_dev/ramdisk/ramdisk.c (hunk 1 above)
Comment (Deryugin) [inline src/drivers/block_dev/ramdisk/ramdisk.c]: It's already done
https://github.com/embox/embox/commit/6cc1c27f369e1d9609e6c69f816f6de54541206c

**Pull request merged**